### PR TITLE
blktrace: unbreak

### PR DESCRIPTION
**Pull request by ndowens**
Did not give any build issue, nor quick run check

### CODE REVIEW

**Comment by maxice8** [comment]
name the commit as: unbreak

**Comment by ndowens** [comment]
> name the commit as: unbreak

Done

**Comment by ndowens** [comment]
Eh seems like removing it causes issues, but will see what I can do.

**Comment by ndowens** [comment]
Sigh, It built on my end :\

File: srcpkgs/blktrace/template [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Template file for 'blktrace'
 pkgname=blktrace
 version=1.2.0
-revision=2
+revision=3
 build_style=gnu-makefile
+make_build_args="-j1"
```
Comment (maxice8) [inline srcpkgs/blktrace/template]: if you need to disable parallel builds use disable_parallel_build

File: srcpkgs/blktrace/template [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Template file for 'blktrace'
 pkgname=blktrace
 version=1.2.0
-revision=2
+revision=3
```
Comment (maxice8) [inline srcpkgs/blktrace/template]: when unbreak it is not necessary to bump revision

**Comment by maxice8** [comment]
Please notify when proposed changes are made

**Pull request merged**